templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -1,5 +1,5 @@
 name: ovs-configuration.service
-enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else}}false{{end}}
+enabled: {{if eq .NetworkType "OVNKubernetes" "OpenShiftSDN"}}true{{else}}false{{end}}
```

Comment:
Is this ever `false` now?

Reply:
CNO also supports kuryr and other 3-party CNIs.